app/assets/javascripts/discourse/app/instance-initializers/hashtag-css-generator.js

```diff
@@ -15,13 +15,6 @@ export default {
   initialize(owner) {
     this.site = owner.lookup("service:site");
 
-    // If the site is login_required and the user is anon there will be no categories
```

Comment:
Do not return early to make sure that hashtags can be generated for the other types of hashtags even if there are no categories loaded.